src/test/java/uk/gov/justice/digital/delius/service/OffenderUpdatesServiceTest.java

```diff
@@ -37,59 +37,59 @@ public void resetRetries() {
 
         @Test
```

Comment:
Looks like we need some unit tests describing that failed updates are retrieved after new created updates